opds.py

```diff
@@ -1071,13 +1071,13 @@ def single_entry(
         if raw or entry is None:
             return entry
         if isinstance(entry, OPDSMessage):
-            entry = unicode(entry)
+            # entry = entry.decode("utf-8")
```

Comment:
What's going on here?

Reply:
This was left over when I was debugging which I'll remove. In some tests, `entry` was a byte string but I resolved that (I just don't remember exactly how).